FastSurferCNN/run_prediction.py

```diff
@@ -169,10 +171,10 @@ def conform_and_save_orig(self, orig_str: str) -> Tuple[nib.analyze.SpatialImage
         # Save input image to standard location
         self.save_img(self.input_img_name, orig_data, orig)
 
-        conform_min = self.vox_size == "auto"
-        if not conf.is_conform(orig, conform_min=conform_min, check_dtype=True, verbose=False):
+        if not conf.is_conform(orig, conform_vox_size=self.vox_size, check_dtype=True, verbose=False,
+                               conform_to_1_threshold=self.conform_to_1_threshold):
```

Comment:
I do not find this flag very descriptive. Would rather set it to conform_to_one_mm, but I guess that is personal preference

Reply:
we can change that to conform_to_1mm_threshold